Lib/fontTools/varLib/mutator.py

```diff
@@ -121,6 +122,35 @@ def instantiateVariableFont(varfont, location, inplace=False):
 		log.info("Building interpolated tables")
 		merger.instantiate()
 
+	for glyph in glyf.glyphs.values():
+		if hasattr(glyph, "program"):
+			instructions = glyph.program.getAssembly()
+			# If GETVARIATION opcode is used in bytecode of any glyph add IDEF
+			addidef = False
```

Comment:
This for loop can be written more succinctly as:
`addidef = any(op.startswith("GETVARIATION") for op in instructions)`.

BTW, any reason to use startswith?

Reply:
Done.
Do you think I should check if op contains GETVARIATION? I thought a GETVARIATION opcode will be on its own line and easy to check for with startsWith.

Reply:
I don't remember what the program data structure looks like. I thought the entire string is equal to the op?

Reply:
Yes the entire string including any comments, which is why I'm checking for startsWith.